src/python/dxpy/workflow_builder.py

```diff
@@ -242,7 +271,7 @@ def _validate_json_for_global_workflow(json_spec, args):
     # TODO: get a confirmation before building a workflow that may be costly
     if 'name' not in json_spec:
         raise WorkflowBuilderException(
-            "dxworkflow.json contains no 'name' field, but it is required to build a global workflow")
+            "Workflow spec contains no 'name' field, but it is required to build a global workflow")
```

Comment:
For apps we use `dxapp.json` ( in src/python/dxpy/scripts/dx_build_app.py). Maybe we should change that as well, just to stay consistent?

Reply:
Yes, I think @mhrvol means the error message is worded that way when building an app from an applet. @YuxinShi0423

Reply:
I have checked `src/python/dxpy/scripts/dx_build_app.py` and all messages containing "dxapp.json" are inside the building process of apps from dxapp.json.

Reply:
But in `app_builder.py` there are some messages that need to be updated. Please check the other comment.